shared/haml/user_header.haml

```diff
@@ -101,8 +101,8 @@
     .header_button.header_user#header_user_signin
       %span= I18n.t("#{loc_prefix}signin")
   -# Hides the Create account button on the sign up page
-  - if request.path_info != '/users/sign_up' && !request.path_info.start_with?('/users/new_sign_up')
-    %a.linktag{href: CDO.studio_url('users/new_sign_up/account_type', ge_region: ge_region), class: 'button-create-account desktop', id: 'create_account_button'}
+  - if !request.path_info.start_with?('/users/sign_up')
+    %a.linktag{href: CDO.studio_url('users/sign_up/account_type', ge_region: ge_region), class: 'button-create-account desktop', id: 'create_account_button'}
```

Comment:
I simplified this since I think having both checks would end up being redundant now that they start with the same text of `/users/sign_up`